src/staticdata_utils.c

```diff
@@ -836,6 +836,7 @@ static void jl_copy_roots(jl_array_t *method_roots_list, uint64_t key)
 // verify that these edges intersect with the same methods as before
 static jl_array_t *jl_verify_edges(jl_array_t *targets, size_t minworld)
 {
+    JL_TIMING(VERIFY_IMAGE, VERIFY_Edges);
```

Comment:
Talking about high frequency zones, the biggest one by far seems to be the lock spinning:

<img width="628" alt="image" src="https://user-images.githubusercontent.com/1282691/234322413-9b5fb564-e31b-4c0a-9cba-9ae8e15b7a22.png">

Reply:
Is that saying the LOCK_SPIN event is hit 2.8 million times? I'd be interested to know why we're hitting locks 2.8 million times in what's presumably a short profiling run.

Reply:
It's not that short. It is a one min run loading OmniPackage.jl

Reply:
That's still roughly 46k locks per second on average, which sounds really high.

Reply:
Most of those have no parent though, which is kind of a shame.

Reply:
Let's add it then :)

Reply:
> These events are pretty high frequency though, so we might want them disabled by default.

I don't think so since this is outside of the loop. For a big package load we only get ~500 event triggered from this.